app/js/domains.json

```diff
@@ -25,6 +25,7 @@
   {"name": "atwhatsapp", "version": "0.9", "registrar": "0xfa6512358d241b636edf5f8ccaaf86676dec21a7"},
   {"name": "atyoutube", "version": "0.9", "registrar": "0xfa6512358d241b636edf5f8ccaaf86676dec21a7"},
   {"name": "buymecoffee", "version": "0.9", "registrar": "0xfa6512358d241b636edf5f8ccaaf86676dec21a7"},
+  {"name": "coinsto", "version": "0.9", "registrar":   "0xc22734f7ff7e44728ebe043fe8626862254756d6"},
```

Comment:
Unless you registered this with the ensnames version of the contract, this should be formatted like the first few lines - eg, version 1.0, no registrar address.